ios/Plugin/Plugin.swift

```diff
@@ -26,6 +29,113 @@ public class CloudSDK: CAPPlugin {
                                                               environment: environment)
 
         PACECloudSDK.shared.setup(with: configuration)
+        AppKit.shared.delegate = self
+
         call.resolve()
     }
+
+    @objc
+    public func listAvailableCoFuStations(_ call: CAPPluginCall) {
+        guard !isCofuStationsFetchRunning else {
+            call.reject("Failed listAvailableCoFuStations due to: Fetch already running...")
+            return
+        }
+
+        guard let countries = call.getArray(Constants.countries, String.self) else {
+            call.reject("Failed listAvailableCoFuStations due to a missing value for '\(Constants.countries)'")
+            return
+        }
+
+        let countryString = countries.filter { !$0.isEmpty }.joined(separator: ",")
+        let request = GeoJSONAPI.GetBetaGeojsonPois.Request(fieldsgasStation: "stationName,brand",
+                                                            filterpoiType: .gasStation,
+                                                            filteronlinePaymentMethod: "paydirekt,paypal,creditcard,sepa,pacePay,applepay",
```

Comment:
This is ok for now, but we should keep in mind that clients might want to set different filters here.